src/index.html

```diff
@@ -1,12 +1,299 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <div class="card__info">
+          <p class="card__item-model">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+        <div class="card__stars card__stars--4">
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <p class="card__text">Reviews: 5</p>
+        </div>
+        <div class="card__price-block">
+          <p class="card__price">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <a href="http://index.html" class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card card--1">
```

Comment:
WHy do you need modifiers for each card? Let's imagine there will be 1000 cards on the page. How will you handle it?